docs/source/internals.rst

```diff
@@ -126,7 +126,7 @@ via agent and resource configuration files (see :ref:`chapter_supported`).
 
 
 Round-Robin Scheduler
----------------------
+----------------------
```

Comment:
I think rst does not like underlines longer than the text...

Reply:
As strange as it is, it complains for them to be too short if I do not add one more dash. Only in this page, no idea why and did not want to spend even more time on that than I had already to do.